terragrunt/env/production/hosted_zone/terragrunt.hcl

```diff
@@ -0,0 +1,7 @@
+include {
+  path = find_in_parent_folders()
+}
+
+terraform {
+  source = "git::https://github.com/cds-snc/url-shortener//terragrunt/aws/hosted_zone?ref=${get_env("INFRASTRUCTURE_VERSION")}"
+}
```

Comment:
Super minor, but just be consistent with the API `terragrunt.hcl` files, could you put the `terraform` block at the top of the file.

This has absolutely no impact on how Terragrunt works, it's just for my brain 😅

Reply:
I totally hear you! Sorry, I did not notice it and you are right - it is super annoying to have different formats in similar files. I have corrected this.

Reply:
Thank you!